docker/jenkins/demo_deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -ex
+
+DEIS_APP_NAME="bedrock-demo-${BRANCH_NAME#demo__}"
+# convert underscores to dashes. Deis does _not_ like underscores.
+DEIS_APP_NAME=$( echo "$DEIS_APP_NAME" | tr "_" "-" )
+DOCKER_IMAGE_TAG="${DEIS_APP_NAME}:${GIT_COMMIT}"
+
+docker tag "bedrock_dev_final:${GIT_COMMIT}" "localhost:5001/$DOCKER_IMAGE_TAG"
+
+echo "Pushing ${DOCKER_IMAGE_TAG} to docker repository"
+docker push "localhost:5001/${DOCKER_IMAGE_TAG}"
```

Comment:
NIT: "localhost:5001" here and a few lines above should be a variable IMO, perhaps REGISTRY?